dnf/artifact-registry.py

```diff
@@ -0,0 +1,60 @@
+#   manpage for plugin(s)
+#   config file
+
+import dnf
+
+import google.auth
+import google.auth.transport.requests
```

Comment:
you might want to consider `from google.auth.transport import requests` (2.2.4 in the style guide).

Reply:
yes normally, but we want to be clear that we are pulling the gat-specific 'requests' package, rather than the well known one. this was in our code examples for usage of this package